services/wallet/thirdparty/cryptocompare/client.go

```diff
@@ -51,7 +51,7 @@ func (c *Client) DoQuery(url string) (*http.Response, error) {
 }
 
 func (c *Client) FetchPrices(symbols []string, currencies []string) (map[string]map[string]float64, error) {
-	chunks := utils.ChunkSymbols(symbols)
+	chunks := utils.ChunkSymbols(symbols, 60)
```

Comment:
why is this 60? just for me to understand

Reply:
there is a limit on the api regarding the number of character, and I counted 5 character per symbol: 5*60 => 300

Reply:
A constant defined at the top would be clearer